server/server.js

```diff
@@ -51,64 +57,95 @@ module.exports = function(__dirname) {
 	app.use(renderer.handle.bind(renderer))
 	app.use(api)
 
+	/**
+	  * Annotations
+	**/ 
+
+	var requireAdmin = (fn) =>
```

Comment:
Why is this not just `function requireAdmin`?  I don't see a scoping concern here.